### ISSUE TITLE
[1.10.2] Altar commands refuse to work

### ISSUE THREAD

**Issue by DestinyPlayer** [jaredlll08/ModTweaker#413]
#### Issue Description:
I try to use the commands for BloodMagic that are in the files. They used to work three ModTweaker versions ago, but now they don't, even though the commands are still the same.

#### What happens:
It gives me an unknown error and doesn't remove the recipes I intended, but it DOES add the Alchemy Table recipes..

#### What you expected to happen:
The altar recipes for the Blood Orbs should be removed, and replaced with Alchemy Table ones.

#### Script used (Please Pastebin or gist your script, posting an unpasted or ungist'd script will automatically close this issue):
http://pastebin.com/QaXfn5qW

#### Minetweaker.log file (Please Pastebin or gist your file, posting an unpasted or ungist'd file will automatically close this issue):
http://pastebin.com/D22DETaU

____
#### Affected Versions (Do *not* use "latest"):

- Minecraft: 1.10.2
- Forge: 12.18.3.2185
- Crafttweaker:  3.0.17
- ModTweaker: 2.0.9

#### Your most recent log file where the issue was present: 
http://pastebin.com/auYkSCbT

**Comment by jaredlll08** [comment]
So the recipes work, the problem is they are not being removed from JEI, I spent the whole day (literally) looking into this, I honestly have no idea why it isn't working. Blood magic does weird stuff with their JEI stuff.

**Comment by BrewAndCode** [comment]
I will confirm the JEI issue.  I just spent an hour adding and tweaking recipes for the alter just to find that all my changes were additive into JEI.  JEI's now reporting like 20 of the same recipes (once for each /mt reload) I did.

crazy.

**Comment by jaredlll08** [comment]
Well don't reload, reload is known to cause problems.

**Comment by BrewAndCode** [comment]
it happens on server/client restarts too.   Your response is basically like; Hey the brakes on the car  don't work!   You: Well don't use them!  Nice. :P

**Comment by jaredlll08** [comment]
No, my response is, the feature that is only used by pack authors (this issue is a problem regardless of reloading), breaks with mods, in your car analogy, it would be like using rocket fuel to fuel a car, sure it runs, but eventually it will explode.

and before you even mentioned "well remove the feature then", the last time I even suggested it, I nearly got burnt alive by modpack authors.

Also reloading isn't a feature of Modtweaker, it is a feature of Minetweaker/Crafttweaker.

**Comment by BrewAndCode** [comment]
no; I wouldn't ever say remove a feature but rather fix the feature. In development once you've exposed an audience to a feature it's typically poor form to remove it unless an alternative has been provided.

I know it's primarily for pack authors; but with Curse/Twitch being what it is these day that audience is growing a lot.

I can't for the life of me figure out where the hell that recipe data is being saved either.

**Comment by jaredlll08** [comment]
Now we reach the actual problem, I CAN NOT fix the issue. The issue is that, when the feature was originally coded, it was never meant to touch modded stuff, it was only meant for vanilla stuff, because that is simple and it works, there is no auto crafting in vanilla, but once you combine the reload functionality with something such as buildcraft, the laser from that, every single tick it checks it's recipe, so if the recipe is now suddenly removed (which happens due to how reloading works), the game crashes. so for me to "fix" all of these issues, it would require me to go through every single mod released (there are about 7512 mods on curse alone), test it with the reload functionality, and then personally make a PR to the mod to fix the issue, which may not even be possible due to what is actually causing the issue or due to the mod being closed source. 

TL;DR there is no fix.

**Comment by bookerthegeek** [comment]
@jaredlll08 So this is on BloodMagic side with their integration? I can go report to them if it is.

**Comment by jaredlll08** [comment]
@bookerthegeek it is our side, I just haven't found the time to fix it

**Comment by bookerthegeek** [comment]
kk
